src/containers/Application/ElementsBox.tsx

```diff
@@ -57,10 +59,9 @@ const ElementsBox: React.FC<ElementsBoxProps> = ({
              * Also considering send to server on 'Next' or adding a Save button.
              **/
             // TODO: Only send mutation on loose focus event.
-            if (isValid)
-              responseMutation({
-                variables: { id: response?.id as number, value: { text: value } },
-              })
+            responseMutation({
+              variables: { id: response?.id as number, value, isValid },
```

Comment:
I've made a comment on an earlier PR from Nicole. I think this mutation should happen inside ApplicationViewWrapper (original plan for this callback was based on assumption that are now invalidated)

Reply:
Yup, done 852bbf1